docs/source/etl/lift-over.rst

```diff
@@ -0,0 +1,114 @@
+=========
+Liftover
+=========
+
+Liftover tools convert genomic data between reference assemblies. The `UCSC liftOver tool`_  uses a `chain file`_ to
+perform simple coordinate conversion, for example on `BED files`_. The `Picard LiftOverVcf tool`_ also uses the new
+`reference assembly file`_ to transform variant information (eg. alleles and INFO fields).
+Glow can be used to run `coordinate liftover`_ and `variant liftover`_.
+
+.. _`UCSC liftOver tool`: https://genome.ucsc.edu/cgi-bin/hgLiftOver
+.. _`chain file`: https://genome.ucsc.edu/goldenPath/help/chain.html
+.. _`reference assembly file`: https://software.broadinstitute.org/gatk/documentation/article?id=11013
+.. _`BED files`: https://genome.ucsc.edu/FAQ/FAQformat.html#format1
+.. _`Picard LiftOverVcf tool`: https://software.broadinstitute.org/gatk/documentation/tooldocs/current/picard_vcf_LiftoverVcf.php
+
+Create a liftover cluster
+==========================
+
+For both coordinate and variant liftover, pull a chain file down to every node in the Apache Spark cluster using an
+:ref:`initialization script <cluster-scoped-init-script>`. The following example downloads a chain file for liftover
```

Comment:
I think this ref is dead? Also, here you could just say that you need a chain file on every node.

Reply:
Fixed